model/pokemon.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+type Pokemon struct {
+	Id             int    `csv:"ID"`
```

Comment:
Initialisms should be all underscored or all caps, i.e.: `ID int `csv:"ID"``

https://github.com/golang/go/wiki/CodeReviewComments#initialisms